docs/docs-book/src/contributing-to-coffee.md

```diff
@@ -0,0 +1,136 @@
+# Contributing to Coffee
+
+## Table of Content
+
+- Introduction
+- Code Style
+- Commit Style
+- How to make the release
+
+## Introduction
+
+Welcome to the HACKING guide and let's peek into how a day in the life of a **Coffee** plugin manager maintainer looks like.
+
+After reading this, you should be prepared to contribute to the repository and be a potential maintainer in the future if you desire!
+
+Before beginning, please review the most frequent mistakes people make when creating patches at https://www.kernel.org/doc/html/next/process/applying-patches.html#common-errors-when-patching.
```

Comment:
```suggestion
Before beginning, please review and understand this guide otherwise your patch may be ignored!
```

This is a change for the next time